src/Tizen.System.SystemSettings/Tizen.System.SystemSettings/SystemSettingsEventArgs.cs

```diff
@@ -1213,4 +1213,37 @@ internal RotaryEventEnabledChangedEventArgs(bool val)
         /// <since_tizen> 6 </since_tizen>
         public bool Value { get; }
     }
+
+    /// <summary>
+    /// This is EventArgs type for the OobeChanged event.
```

Comment:
To make this clickable link in API reference you can write it like this:
```suggestion
    /// This is EventArgs type for the <see cref="OobeChanged"/> event.
```